modules/apps/portal-odata/portal-odata-impl/src/test/java/com/liferay/portal/odata/internal/filter/ExpressionVisitorImplTest.java

```diff
@@ -507,6 +507,32 @@ public void testVisitBinaryExpressionOperationWithOrOperation() {
 			queryBooleanClause2.getBooleanClauseOccur());
 	}
 
+	@Test
+	public void testVisitBinaryExpressionWithSuboperation() {
```

Comment:
This name doesn't follow the same pattern that the rest of the name of the test class.

Reply:
and the same for this method : testVisitBinaryExpressionOperationWithSuboperationVisitingLiteralExpressions

Reply:
@cgoncas